frontend/src/components/MyNavbar/MyNavbar.js

```diff
@@ -0,0 +1,84 @@
+import React, {useState} from 'react'
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './MyNavbar.css'
+import {
+    Navbar,
+    Button,
+    ButtonDropdown,
+    DropdownItem,
+    DropdownMenu,
+    DropdownToggle
+  } from 'reactstrap';
+
+  import ButtonSpacer from '../ButtonSpacer/ButtonSpacer'
+
+
+const LinkButton = (props) => {
+    return(
+        <>
+        <Button color="info" id="button-background" href = {props.url}>
+           <text className="button-text">{props.label}</text>
+        </Button>
+        <ButtonSpacer/>
+        </>
+    )
+}
+
+const DropButton = (props) => {
+
+    const [dropdownOpen, setOpen] = useState(false);
+    const toggle = () => setOpen(!dropdownOpen);
+
+    return(
+        <ButtonDropdown color="info" id="button-background" isOpen={dropdownOpen} toggle={toggle}>
+                <DropdownToggle color="info" caret>
+                <text className="button-text">Info</text>
+                </DropdownToggle>
+                <DropdownMenu>
+                    <DropdownItem href="/team"> The Team </DropdownItem>
+                    <DropdownItem href="/tools"> Tools </DropdownItem>
```

Comment:
I love that you figured out how to use the functionality of the dropdown but since the tools and team components will be displayed on the landingpage, im not sure it will be very useful here. What do you think?

Reply:
I agree. I put it in just in case since I was not sure where the components would go. It has been removed.